release_notes.md

```diff
@@ -1,11 +1,48 @@
 # Gazebo Ionic Release Notes
 
-## New Features
+## gazebosim/gz-utils:
 
-## Bug Fixes
+- [Add Logging utility class based on `spdlog`](https://github.com/gazebosim/gz-utils/pull/134).
+  Also see [gz-utils#145](https://github.com/gazebosim/gz-utils/pull/145),
+  [gz-utils#144](https://github.com/gazebosim/gz-utils/pull/144),
+  [gz-utils#142](https://github.com/gazebosim/gz-utils/pull/142),
+  [gz-utils#141](https://github.com/gazebosim/gz-utils/pull/141),
+  [gz-utils#139](https://github.com/gazebosim/gz-utils/pull/139)
+- [Add new functions for manipulating the environment](https://github.com/gazebosim/gz-utils/pull/114)
+- [Make the single argument constructor of `Subprocess` inherit the env](https://github.com/gazebosim/gz-utils/pull/113)
+- [bazel: build and test subprocess functionality](https://github.com/gazebosim/gz-utils/pull/123)
+- [bazel:Add license checking support](https://github.com/gazebosim/gz-utils/pull/108)
+- [Add package.xml](https://github.com/gazebosim/gz-utils/pull/125)
+- [Require cmake version 3.22.1](https://github.com/gazebosim/gz-utils/pull/132)
 
-## Breaking Changes
 
-## Documentation
+## gazebosim/gz-physics:
```

Comment:
what is the ordering strategy for the package release notes? it doesn't seem to be topological or alphabetical

Reply:
I'm thinking alphabetical since it's the easiest.

Reply:
I think that's what did for Harmonic https://github.com/gazebosim/gz-harmonic/blob/main/release_notes.md